homework-7/ex-1_chat/index.html

```diff
@@ -0,0 +1,179 @@
+                        </td>
+                    </tr>
+                    <tr>
+                        <td class="avatar">
+                            <img src="./imgs/erica-hughes .jpg" alt="">
+                        </td>
+                        <td>
+                            <h4>Henry Lau</h4>
+                            <span><i class="fa fa-circle" style="color: green;"></i>&nbsp online</span>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td class="avatar">
+                            <img src="./imgs/ginger-johnston .jpg" alt="">
+                        </td>
+                        <td>
+                            <h4>Lee Jong Suk</h4>
+                            <span><i class="fa fa-circle" style="color:rgb(196, 137, 90);"></i>&nbsp left 30 mins ago</span>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td class="avatar">
+                            <img src="./imgs/peyton-mckinney .jpg" alt="">
+                        </td>
+                        <td>
+                            <h4>Vincent Porter</h4>
+                            <span><i class="fa fa-circle" style="color: green;"></i>&nbsp online</span>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td class="avatar">
+                            <img src="./imgs/mike-thomas.jpg" alt="">
+                        </td>
+                        <td>
+                            <h4>Vincent Porter</h4>
+                            <span><i class="fa fa-circle" style="color: green;"></i>&nbsp online</span>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td class="avatar">
+                            <img src="./imgs/erica-hughes .jpg" alt="">
+                        </td>
+                        <td>
+                            <h4>Erica hughes</h4>
+                            <span><i class="fa fa-circle" style="color: rgb(129, 126, 109);"></i>&nbsp offline</span>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td class="avatar">
+                            <img src="./imgs/christian-kelly .jpg" alt="">
+                        </td>
+                        <td>
+                            <h4>Vincent Porter</h4>
+                            <span><i class="fa fa-circle" style="color: rgb(196, 137, 90);"></i>&nbsp left 1 day ago</span>
+                        </td>
+                    </tr>
+                </table>
+            </div>
+
+            <div class="Mess">
+                <div class="Mess__profile">
+                    <div class="Mess--avatar">
+                        <img src="./imgs/christian-kelly .jpg" alt="">
+                    </div>
+                    <div class="Mess--name">
```

Comment:
```suggestion
                    <div class="mess--name">
```
Trong HTML CSS tất cả phải viết thường